operator/Makefile

```diff
@@ -1,3 +1,4 @@
+# Meaningless change to get CI to run ocp tests. Remove before merging.
```

Comment:
Is it a good time to remove this line already? You can manually trigger operator tests with `/test ...` command, just get the full list with `/test ?`.

Reply:
Removed